.github/workflows/create-release-tag-and-pr.yml

```
@@ -45,9 +45,29 @@ jobs:
 
       # Create a new branch off the release branch for the PR, we do this to ensure the release branch
       # is not affected
+
+      - name: Setup Node.js
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
+        with:
+          node-version: '20.x'
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn install
+
       - name: Create branch
         run: |
           git checkout -b chore/${{ github.event.inputs.tag }}-release
+
+      - name: Generate CHANGELOG.md
+        run: |
+          node ./scripts/get-changelog.js -f ${{ github.event.inputs.previous-tag }} -t ${{ github.event.inputs.tag }} -w true
+
+      - name: Commit changelog
+        run: |
+          git add CHANGELOG.md
```

```suggestion
          git add .
```

I think this should actually be `git add .` or maybe `git add '**/CHANGELOG.md'` to scope it down.